signer/gpg2/gpg2_test.go

```diff
@@ -529,7 +541,7 @@ func TestGPG2Signer_SignFiles(t *testing.T) {
 		})
 	}
 
-	matches, err := filepath.Glob("/tmp/autograph_*sign_files*")
+	matches, err := filepath.Glob("/tmp/autograph_*sign_files*/*")
```

Comment:
Since we're doing things in parallel, I think we could still accidentally hit this if another test is running that's writing this same pattern, no? Unless we pass in the full dir path